src/main/java/core/basesyntax/model/Fruit.java

```diff
@@ -0,0 +1,64 @@
+package core.basesyntax.model;
+
+public class Fruit {
```

Comment:
do we need this class? we have the same fields, getters and setters in FruitTransaction